scenarios/aws/microVMs/microvms/run.go

```diff
@@ -73,7 +73,7 @@ func newEC2Instance(awsEnv aws.Environment, name, ami, arch, instanceType, keyPa
 
 	instance, err := awsEc2.NewInstance(awsEnv.Ctx, awsEnv.Namer.ResourceName(name), &awsEc2.InstanceArgs{
 		Ami:                 pulumi.StringPtr(ami),
-		SubnetId:            pulumi.StringPtr(awsEnv.DefaultSubnets()[0]),
+		SubnetId:            awsEnv.RandomSubnets().Index(pulumi.Int(0)),
```

Comment:
I think we would like to opt out of randomly selecting the subnets. We did some coarse experimentation to see which AZs had the most capacity for the resources we want, and we are selecting the subnets based on that.

Reply:
You can "opt-out" if you pass only 1 subnet, which, I think, is what you do?